org.d3cod3.Mosaic.yml

```diff
@@ -0,0 +1,446 @@
+        # branch: master
+        # tag:
+        commit: 08b0ad91113f17f4eb276441adfedb54869bdc9b
+        dest: addons/ofxFontStash
+      - type: git
+        url: https://github.com/Akira-Hayasaka/ofxGLEditor.git
+        # branch: master
+        # tag:
+        commit: 830aac727a688ccd87de780f6f9d82e8ab2ffc4b
+        dest: addons/ofxGLEditor
+      - type: git
+        url: https://github.com/jeffcrouse/ofxJSON.git
+        # branch: master
+        # tag:
+        commit: 55a5825ddf510c991466f9d39d8e1d57b2bc2008
+        dest: addons/ofxJSON
+      - type: git
+        url: https://github.com/d3cod3/ofxImGui.git
+        # branch: master
+        tag: 0.4.8
+        commit: 73cf7f142958c7588c82d5ddd59aba2b556e3daa
+        dest: addons/ofxImGui
+      - type: git
+        url: https://github.com/d3cod3/ofxInfiniteCanvas.git
+        # branch: master
+        tag: 0.4.8
+        commit: 810664d7046901f3fb6b5d5627f618342acf7776
+        dest: addons/ofxInfiniteCanvas
+      - type: git
+        url: https://github.com/d3cod3/ofxLua.git
+        tag: 0.4.8
+        commit: c5fd0858de700e1b85ff2dffd7d005b4c691e6d9
+        dest: addons/ofxLua
+      - type: git
+        url: https://github.com/danomatika/ofxMidi.git
+        # branch: master
+        # tag: 0.4.8
+        commit: e44c14a1fc76fe33786cb21f2fd908d0b4217605
+        dest: addons/ofxMidi
+      - type: git
+        url: https://github.com/d3cod3/ofxMtlMapping2D.git
+        # branch: master
+        tag: 0.4.8
+        commit: fcb4144231a2aaa9aae27cb5f8e98ba940d86839
+        dest: addons/ofxMtlMapping2D
+      - type: git
+        url: https://github.com/d3cod3/ofxNDI.git
+        # branch: master
+        tag: 0.4.8
+        commit: a777204469729eb3760f780b2da4b20080245f50
+        dest: addons/ofxNDI
+      - type: git
+        url: https://github.com/danomatika/ofxPd.git
+        # branch: master
+        # tag:
+        commit: 9ab7b10a5e69b101ae687587bb75007989ff0b18
+        dest: addons/ofxPd
+      - type: git
+        url: https://github.com/d3cod3/ofxPdExternals.git
+        # branch: master
+        tag: 0.4.8
+        commit: 57a4a36b0bf361d943ce5ef4621b26816ab2e08f
+        dest: addons/ofxPdExternals
+      - type: git
+        url: https://github.com/npisanti/ofxPDSP.git
+        # branch: master
+        # tag:
+        commit: e106991f4abf4314116d4e7c4ef7ad69d6ca005f
+        dest: addons/ofxPDSP
+      - type: git
+        url: https://github.com/d3cod3/ofxPython.git
+        # branch: master
+        tag: 0.4.8
+        commit: 606fb3538e396539540bbb90317bfc9566d729a4
+        dest: addons/ofxPython
+      - type: git
+        url: https://github.com/d3cod3/ofxTimeline.git
+        # branch: master
+        tag: 0.4.8
+        commit: 2fc353fb45f6d659db693f13929976be664a02e7
+        dest: addons/ofxTimeline
+      - type: git
+        url: https://github.com/d3cod3/ofxVisualProgramming.git
+        # branch: master
+        tag:
```

Comment:
empty tag is invalid.

Reply:
I was waiting to the releases of ofxVisualProgramming and Mosaic. Solved now